package.json

```diff
@@ -40,6 +40,8 @@
     "js2xmlparser2": "^0.2.0",
     "lodash": "^4.0.0",
     "node-simctl": "^3.1.0",
+    "request": "^2.79.0",
+    "request-promise": "^2.0.1",
```

Comment:
Weird. That was just the result of `npm -S request-promise`, so it should have gotten the latest.